ckanext/depositar_theme/fanstatic/scss/_others.scss

```diff
@@ -1,10 +1,45 @@
+body {
+  line-height: 30px;
+}
+
+:focus {
+  outline: 0;
+}
+
+p {
+  margin-top: revert;
+}
+
+h1 {
+  font-weight: bold;
+}
```

Comment:
Please limit the scope for the CSS selectors of the styles from the landing page if needed since they may accidentally affect the other pages. For example, the style for the `p` element makes the `ark` and `other access` widget look weird. And, it seems that the style for the `:focus` selector is no longer needed in the BS 5 theme.